src/ChainSafe.Gaming/Web3/Core/Environment/IOperatingSystemMediator.cs

```diff
@@ -15,6 +15,8 @@ public interface IOperatingSystemMediator
 
         public Platform Platform { get; }
 
+        public string ApplicationDataPath { get; }
```

Comment:
Is this the same as `Application.PersistentDataPath` if so it should be renamed to `ApplicationPersistentDataPath` because `Application.dataPath` is a path to `Assets` folder in Unity and this would be confusing in that context

Reply:
What about `AppPersistentDataPath`?

Reply:
Yeah that works